### ISSUE TITLE
[DiceSoNice] Wrong results displayed on 3D dice compared to the chat

### ISSUE THREAD

**Issue by Stefouch** [fvtt-fria-ligan/twilight2000-foundry-vtt#120]
### Describe the bug

See https://gitlab.com/riccisi/foundryvtt-dice-so-nice/-/issues/353

### Reproduction

Roll "Ranged Combat" with Rate of Fire at max

### System Info

```sh
System: 10.0.1
Foundry: 10.290
DsN: 4.6.5
OS: Win 10, client
```

### Priority this issue should have?

High (a broken feature, major functional impact)

### Validations

- [X] Done a clean install of the system.
- [X] Disabled all modules.
- [X] Checked if the issue is present in a new world instance (create a new world in foundry using the Twilight 2000 system).

**Comment by Stefouch** [comment]
Confirmed: it's a bug from Dice So Nice.

The dev said it hould be fixed soon!

**Comment by Stefouch** [comment]
Should be fixed with version 4.6.6 of DsN
